### PR TITLE
Orphan processes

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#34 
#### What does this implement/fix? Explain your changes.
- `NavaThread.stop` method bug fixed

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
## Local tests on OSs
- [ ] MacOS Sonoma 14.2.1
- [x] Windows 10
- [ ] Ubuntu 16.04
- [ ] Ubuntu 22.04

File: test/function_test.py [hunk 1]
```diff
@@ -14,6 +15,10 @@
 True
 >>> stop(1001)
 >>> stop_all()
+>>> for i in range(40):
+...     sound_id = play(os.path.join("others", "test.wav"), async_mode=True)
+...     time.sleep(0.5)
+...     stop(sound_id)
```
Comment (sadrasabouri) [inline test/function_test.py:21]: Is `stop` here necessary for test?

**Review by sadrasabouri — COMMENTED** [review]
I tested at MacOS both the package performance and `nafas` pytest (as a depended) are OK, I will test on Ubuntu in less than 12 hours.

**Comment by sadrasabouri** [comment]
I tested at Ubuntu 22.04 both the package performance and `nafas` pytest (as a depended) are OK.

File: test/function_test.py (hunk 1 above)
Comment (sadrasabouri) [reply test/function_test.py:21]: This one will be fixed in future PR for handling tests.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**